extensions/replication/tasks/ReplicateObject.js

```diff
@@ -363,25 +397,36 @@ class ReplicateObject extends BackbeatTask {
                 // eslint-disable-next-line no-param-reassign
                 err.retryable = true;
                 log.error('an error occurred when streaming data from S3',
-                          { method: 'ReplicateObject._getAndPutPartOnce',
-                            entry: destEntry.getLogInfo(),
-                            part,
-                            origin: 'source',
-                            peer: this.sourceConfig.s3,
-                            error: err.message });
+                    {
+                        method: 'ReplicateObject._getAndPutPartOnce',
+                        entry: destEntry.getLogInfo(),
+                        part,
+                        origin: 'source',
+                        peer: this.sourceConfig.s3,
+                        error: err.message,
+                    });
             }
             return doneOnce(err);
         });
+        incomingMsg.on('end', () => {
+            this.metricsHandler.latency({
```

Comment:
"latency" might not be the best term for measuring the time to completion of an operation, it would rather be referring to the time to first byte in such case. I think renaming to "elapsed", "elapsedMs" or other similar name would be more accurately reflecting what it measures.